doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -837,6 +840,170 @@ The major differences visible to users are:
+      central repository.
+
+**implementation**
+    How to implement the steps responsible for transferring datasets back to
+    the original Butler repository:
+
+    * JOB: Single bash script is written with sequence of commands and is
+      represented in the GenericWorkflow as a GenericWorkflowJob.
+    * WORKFLOW: (Not implemented yet): Instead of a bash script, make a
+      little workflow representing the sequence of commands.
+
+**concurrency_limit**
+    Name of the concurrency limit. For butler repositories that need to limit
+    the number of simultaneous merges, this name tells the plugin to limit the
+    number of ``finalJob`` jobs via some mechanism, e.g., a special
+    queue.
+
+    * db_limit: special concurrency limit to be used when limiting number of
+      simultaneous butler database connections.
+
+**command1, command2, ...**
+    Commands executed in numerical order as part of the ``finalJob`` job.
+
+
+You can include other job specific requirements in ``finalJob`` section as
+well. For example, to ensure that the job running the quantum-backed Butler will
+have 4 GB of memory at its disposal, use ``requestMemory`` option:
+
+.. code-block:: YAML
+
+   finalJob:
+     requestMemory: 4096
+     ...
+
+Automatic memory scaling (for a WMS plugin that supports it) can be enabled in
+a similar way, for example
+
+.. code-block:: YAML
+
+   finalJob:
+     requestMemory: 4096
+     requestMemoryMax: 16384
+     memoryMultiplier: 2.0
+     ...
+
+User-visible Changes
+^^^^^^^^^^^^^^^^^^^^
+
+The major differences to users are:
+
+* Quantum-backed Butler takes precedence over the execution Butler.
+  If the BPS configuration file contains *both* the ``executionButler`` and the
+  ``finalJob`` sections, the quantum-backed Butler will be used during the
+  workflow execution.
+
+* **bps report** shows a new job called **finalJob** in the detailed view.
+  This job is responsible for transferring datasets from the quantum graph back
+  to the central Butler. Similarly to other jobs, it can succeed or fail.
+
+* Extra files in the submit directory:
+
+  - ``final_job.bash``: Script that is executed to transfer the datasets back to
+    the central repo.
+  - ``quantumGraphUpdate.out``: Output of the command responsible for updating
+    the output run in the provided pre-existing quantum graph.
+  - ``final_post_finalJob.out``: An internal file for debugging incorrect
+    reporting of final run status.
+  - ``<qgraph_filename>_orig.qgraph``: A backup copy of the original
+    pre-existing quantum graph file that was used for submitting the run.  Note
+    that this file will *not* be present in the submit directory if the
+    pipeline YAML specification was used during the submission instead.
+
+.. note::
+
+   For your convenience an example BPS config file the quantum-backed Butler
+   was added to the package,
+   ``$CTRL_BPS_DIR/python/lsst/ctrl/bps/config/bps_qbb.yaml``. You can include
```

Comment:
We should really not be relying on the environment variable but using `resource` URIs as above.